x-pack/platform/test/onechat_functional/tests/tools/create_tool.ts

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { FtrProviderContext } from '../../../functional/ftr_provider_context';
+
+const APP_ID = 'agent_builder';
+
+export default function ({ getPageObjects, getService }: FtrProviderContext) {
+  const { common } = getPageObjects(['common']);
+  const testSubjects = getService('testSubjects');
+  const monacoEditor = getService('monacoEditor');
+  const es = getService('es');
+
+  describe('create tool', function () {
+    let testIndexName: string;
+
+    before(async () => {
+      testIndexName = `ftr_onechat_${Date.now()}`;
+      await es.indices.create({ index: testIndexName });
+      await es.index({
+        index: testIndexName,
+        document: { message: 'hello world', numeric: 1, '@timestamp': new Date().toISOString() },
+      });
+      await es.index({
+        index: testIndexName,
+        document: { message: 'second doc', numeric: 2, '@timestamp': new Date().toISOString() },
+      });
+      await es.indices.refresh({ index: testIndexName });
```

Comment:
I would be careful if you rely on this index to exist with this data in your test you might see flakiness in serverless MKI since it can take up to 30s for new data to be available to query.